pkg/apis/application/v1alpha1/applicationset_types.go

```diff
@@ -182,8 +182,9 @@ func (g ApplicationSetTerminalGenerators) toApplicationSetNestedGenerators() []A
 
 // ListGenerator include items info
 type ListGenerator struct {
-	Elements []apiextensionsv1.JSON `json:"elements" protobuf:"bytes,1,name=elements"`
-	Template ApplicationSetTemplate `json:"template,omitempty" protobuf:"bytes,2,name=template"`
+	Elements     []apiextensionsv1.JSON `json:"elements" protobuf:"bytes,1,name=elements"`
+	ElementsYaml string                 `json:"elementsYaml,omitempty" protobuf:"bytes,2,opt,name=elementsYaml"`
```

Comment:
@lsoica @crenshaw-dev Just FYI, I think this is going to be a breaking change for existing API clients as it changes the meaning of field `2` in the generated protobuf.